src/game/client/tf/vgui/panels/tf_loadoutpanel.cpp

```diff
@@ -483,7 +466,8 @@ void CTFLoadoutPanel::UpdateModelWeapons( void )
 void CTFLoadoutPanel::Show()
 {
 	BaseClass::Show();
-	MAINMENU_ROOT->ShowPanel( SHADEBACKGROUND_MENU );
+	CTFLoadoutPanel *m_bShadedBG = this;
```

Comment:
What's the point of this?

Reply:
I don't remember at all, probably something I had momentarily been messing around with. And changed it to that without thinking